### PR TITLE
Improve disconnection/reconnection UX

### LINKED ISSUE #711: Improve connection lost UX
**Is your feature request related to a problem? Please describe.**

When the card cannot communicate with HA for fetching the JSMPEG stream, it displays a message of:

```
Could not retrieve or sign JSMPEG websocket path. [Check troubleshooting](https://github.com/dermotduffy/frigate-hass-card#troubleshooting).
```

But when HA backend is simply unavailable (like during restarts), the error message above isn't helpful (and a little ugly).

chrome_dmVHWodBnx

**Describe the solution you'd like**
Home Assistant frontend can properly detect whether connection to backend was lost or not (that's why it shows a `Connection lost. Reconnecting...` toast message below).

If we could reuse that part of the frontend for detecting such a situation, we could improve the error message (by not displaying an error at all).

How about showing a "Connection"-like icon like `mdi:connection`, centralized:

image

Another suggestion, which perhaps should be tracked in a different issue, would be to keep the aspect ratio of 16:9 (if the user has defined such a 16:9 fixed aspect ratio) even in such scenarios to avoid breaking the screen layout.

**Describe alternatives you've considered**

N/A.

**Additional context**

What do you think? Happy to hear opinions.

### PR DESCRIPTION
**Pull request by dermotduffy**
* Closes #711

### CODE REVIEW

File: .vscode/i18n-ally-reviews.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Review comments generated by i18n-ally. Please commit this file.
+
+reviews:
+  error.reconnecting:
+    locales:
+      pt-BR:
+        translation_candidate:
+          source: en
+          text: Reconectando
+          time: '2022-06-23T03:45:39.286Z'
```
Comment (felipecrs) [inline .vscode/i18n-ally-reviews.yml:10]: Approved :)

**Comment by felipecrs** [comment]
For some reason, I can't trigger the new message:

https://user-images.githubusercontent.com/29582865/175288482-a8582b7a-a5cf-440b-878f-9815070f71f2.mp4

Am I doing something unexpected?

**Comment by dermotduffy** [comment]
Random idea: try your restart in another window so the card is already running before the restart. This code reacts to a change in the connection state. We may want to change it to also show that message if it's disconnected on first load.

**Comment by felipecrs** [comment]
Thank you, now I was able to see it. Here is how it looks in my dashboard, btw:

https://user-images.githubusercontent.com/29582865/175318410-f87bc9dd-e3b1-4f24-b57f-05e1d5463a06.mp4

For some weird reason the icon wasn't displayed (although I confirmed my frontend contains the icon  by looking for it in an entity configuration).

Another thing: I think it may be worth to hide every custom element (which is not in the menu bar) for displaying such a message.

**Comment by felipecrs** [comment]
About the icon, I will do some research later. My initial analysis concluded that the icon name was correct and not missing though.

About hiding the elements, IMO this falls into your second thought: harder, and probably unnecessary.

**Comment by dermotduffy** [comment]
Just to clarify, if you had to pick: make elements always visible over messages (including custom icons on the menu), or never visible (custom element icons wouldn't be in the menu when there's a message), which would you favor?

**Comment by felipecrs** [comment]
Custom elements never visible.

**Comment by felipecrs** [comment]
https://user-images.githubusercontent.com/29582865/175457649-e7873dd7-e9d1-473f-b908-ca37ef449fe5.mp4

The icon issue is definitely something related to the HA frontend. Maybe it was fixed in newer versions, since I suppose you are using HA beta. But it's fine, no worries, as this is not an issue with the card.

So, here are my remaining suggestions:

1. Make the connection check also during startup
2. Always hide custom elements on displaying messages

**Pull request merged**